lib/trento/application/projectors/sap_system_projector.ex

```diff
@@ -221,6 +241,27 @@ defmodule Trento.SapSystemProjector do
     )
   end
 
+  @impl true
+  def after_update(
+        %ApplicationInstanceDeregistered{
+          instance_number: instance_number,
+          host_id: host_id,
+          sap_system_id: sap_system_id
+        },
+        _,
+        _
+      ) do
+    TrentoWeb.Endpoint.broadcast(
+      @sap_systems_topic,
+      "application_instance_deregistered",
+      SapSystemView.render("application_instance_deregistered.json",
+        id: sap_system_id,
```

Comment:
Let's be careful with the payload.
I think we use `sap_system_id` to identify the sap system in the app/db instances.
If we send it with the same field name, updating the entries in redux will be easier as well.
You can do the transformation in the view if you want